.github/workflows/benchmark-runner.yml

```diff
@@ -0,0 +1,164 @@
+name: Run RediSearch Benchmarks
+
+# Documentation: https://redislabs.atlassian.net/wiki/spaces/DX/pages/3967844669/RediSearch+CI+refactor
+
+on:
+  workflow_dispatch:
+    inputs:
+      extended:
+        type: boolean
+        description: 'Run extended benchmarks'
+        default: false
+  workflow_call:
+    inputs:
+      extended:
+        type: boolean
+        default: false
+
+jobs:
+  benchmark-search-oss-standalone:
+    strategy:
+      matrix:
+        member_id: [1, 2, 3]
+    uses: ./.github/workflows/benchmark-flow.yml
+    secrets: inherit
+    with:
+      benchmark_glob: "search*.yml"
+      benchmark_runner_group_member_id: ${{ matrix.member_id }}
+      benchmark_runner_group_total: ${{ strategy.job-total }}
```

Comment:
Where does `job-total` defined?

Reply:
internally, it holds the number of entries in the matrix. This way if we want to split the work between more machines, you just have to resize the `member_id` list